client/jetpack-cloud/sections/agency-dashboard/sites-overview/hooks/use-install-boost.tsx

```diff
@@ -17,6 +18,8 @@ export default function useInstallBoost(
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 	const queryClient = useQueryClient();
+	const agency = useSelector( getActiveAgency );
+	const agencyId = agency ? agency.id : undefined;
```

Comment:
I think it is time to move this check into its own selector because I'm seeing it more and more in different places. I'm afraid that in the future, we will change how we get the "active" agency ID.

Reply:
Yes, I agree. A maintenance task, right?

Reply:
@rcanepa I created this issue [Create a selector for getting the agencyId](https://github.com/Automattic/automattic-for-agencies-dev/issues/207). I set the priority medium-low that means that we could address it in this project if we have time; if not, it will be moved as a maintenance task.